File: lapack-jni/check/check.scala
import scala.offheap.numeric.jni.LapackJNI;

object main {
  def main (argv: Array[String]): Unit =  {
    LapackJNI.cblas_dgemm(102, 'n', 'n', 2, 2, 2, 1.0, 0, 2, 0, 2, 1.0, 0, 2)
  }
}

File: lapack-jni/check/check.java
import scala.offheap.numeric.jni.LapackJNI;

public class check {
	public static void main (String argv[]) {
		LapackJNI.cblas_dgemm(102, 'n', 'n', 2, 2, 2, 1.0, 0, 2, 0, 2, 1.0, 0, 2);
	}
}
